mqtt/mqtt-bridge/src/controller/mod.rs

```diff
@@ -82,9 +82,13 @@ impl Sidecar for BridgeController {
         }
 
         loop {
-            let wait_bridge_or_pending = if bridges.is_terminated() {
-                // if no active bridges available, wait only for a new messages arrival
-                Either::Left(future::pending())
+            let wait_bridge_or_pending = if (bridges.is_terminated() || bridges.is_empty()) {
```

Comment:
not sure I understood the change. it used to be `future::pending` to make the sidecar running until `BridgeControllerMessage::Shutdown` received in case all bridges stopped.
now with your change, it is ready with some value right away which should cause the loop consuming CPU without a good reason. could you please verify it doesn't behave in the way I described?